db/migrate/20221020131646_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] The likes table should have an `author_id` column, not a `user_id`. And so these references should be `author`, not `user`.